pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
```

Comment:
This is a little confusing to me.  The below commands take a value, which is perfectly intuitive, but they don't take anything like a key.  What's this stuff about "database path"?

Reply:
dconf is an internal settings type of thing, you have probably used it, I don't know.

However it is a big database of internal settings

Reply:
The path is the key. Basically like the registry on Windows

Reply:
In `/example/dconf/path`, is "path" what I'm thinking the "key" is?

>you have probably used it

🙂 Nope, never.  I've never actually even used Linux as my personal desktop, _ever_! 😄

Reply:
> In /example/dconf/path, is "path" what I'm thinking the "key" is?

Exactly :+1:

Reply:
What OS do you use?
I would recommend Linux, much better for programming

Reply:
I have a Mac, so I use macOS Big Sur 11.3 :)

I know Linux is great, but I've always liked Apple 

I have Windows and Linux VMs, so it's not as if I don't know them.  I'm really familiar with both, to tell the truth, but `dconf` I don't recall using ever. 🤔